pages/common/cloudflared.md

```diff
@@ -0,0 +1,20 @@
+# cloudflared
+
+> Command line tool to create a persistent connection to the Cloudflare network.
+> More information: <https://developers.cloudflare.com/argo-tunnel/>.
+
+- Authenticate and associate the connection to a domain in the Cloudflare account:
+
+`cloudflared tunnel login`
+
+- Establish a tunnel between the local server to a host in Cloudflare:
+
+`cloudflared tunnel --hostname {{hostname}} localhost:{{port_number}} --no-tls-verify`
```

Comment:
The `--no-tls-verify` here sounds somewhat dangerous?

Reply:
That would be fine. Most of the time, a web server used for development locally does not have TLS certificate, or use a self-signed certificate.

Reply:
I see. Would it be possible to include an extra example showing how it would be done if you _did_ want to verify the local server's tls certificate please? We should always try to promote good security practices where possible.